ironfish/src/network/peers/peerManager.ts

```diff
@@ -978,57 +979,23 @@ export class PeerManager {
       return
     }
 
-    const identity = message.identity
-    const version = message.version
-    const agent = message.agent
-    const port = message.port
-    const name = message.name
-    const networkId = message.networkId
-    const genesisBlockHash = message.genesisBlockHash
+    const { identity, version, agent, port, name, networkId, genesisBlockHash } = message
```

Comment:
Why do we detatch all these properties from the message? Is it for typescript proving system? `message.identity` is much clearer than {identity}